mps_shape_completion/CMakeLists.txt

```diff
@@ -194,3 +196,9 @@ include_directories(
 
 ## Add folders to be run by python nosetests
 # catkin_add_nosetests(test)
+
+# if (CATKIN_ENABLE_TETING)
+  # find_package(invalid REQUIRED)
+  find_package(rostest REQUIRED)
+  add_rostest(tests/demo_test.test)
```

Comment:
I'm a big fan of roslaunch tests as well:
```
find_package(roslaunch REQUIRED)
roslaunch_add_file_check(tests/demo_tests.test USE_TEST_DEPENDENCIES)
```
(I'm pretty sure it's smart enough to understand `.test` files.)